README.md

```diff
@@ -13,34 +13,33 @@ Currently supports
  - Coub
  - Wistia
 
-# grunt
-
-```shell
-$ grunt
```

Comment:
How do I build the dist locally?
```shell
npm
npm test
npm ?
```

Reply:
Update your local version of node to at least version 6. I'm on version 8. The code uses two ES2015 features: `const` and destructing.

npm should also be updated when you update node. npm updates your `package-lock.json` file automatically when you do npm commands. It lists versions that were actually installed so a build can be exactly reproduced. In contrast package.json lists package version ranges, which allows newer versions to be installed. Not to say that you can't install newer versions of dependencies, but package lock gives you a way to install exact versions. It helps with "but it works for me" situations. Very handy.